tests/k8s/networkpolicy-ingress-allow-namespace.yaml

```diff
@@ -2,8 +2,18 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: skydive-test-networkpolicy-ingress-allow-namespace-to
-  labels:
-    app: skydive-test-networkpolicy-ingress-allow-namespace-to
+---
```

Comment:
going to be changed due to: https://github.com/skydive-project/skydive/pull/1318